product_docs/docs/mongo_data_adapter/5/06_features_of_mongo_fdw.mdx

```diff
@@ -32,6 +34,19 @@ MongoDB Foreign Data Wrapper supports aggregate pushdown. It pushes the aggregat
 
 For more information, see [Example: Aggregate pushdown](08b_example_aggregate_pushdown).
 
+## ORDER BY pushdown
+
+MongoDB Foreign Data Wrapper supports order by push-down. If possible, push order by clause to the remote server so that we get the ordered result set from the foreign server itself. It might help us to have an efficient merge join. NULLs behavior is opposite on the MongoDB server. Thus to get an equivalent result, we can only push down ORDERBY with either ASC NULLS FIRST or DESC NULLS LAST. Moreover, as MongoDB sorts only on
+fields, only column names in ORDER BY expressions are pushed down.
+
+- [Example: ORDER BY pushdown](example_order_by_push_down)
+
+## LIMIT OFFSET pushdown
+
+MongoDB Foreign Data Wrapper supports limit offset push-down. Wherever possible, perform LIMIT and OFFSET operations on the remote server. This reduces network traffic between local PostgreSQL and remote MongoDB servers.
+
+- [Example: LIMIT OFFSET pushdown](example_limit_offset_push_down)
```

Comment:
don't use a bulleted list here or in the ORDER BY section

Reply:
changes done

Reply:
changes done